app/controllers/TagManagementApi.scala

```diff
@@ -37,50 +39,54 @@ object TagManagementApi extends Controller with PanDomainAuthActions {
     }
   }
 
-  def createTag() = (APIAuthAction andThen CreateTagPermissionsCheck).async { req =>
-    implicit val username = Option(req.user.email)
-    req.body.asJson.map { json =>
-      json.as[CreateTagCommand].process.map { result =>
-        result.map{t => Ok(Json.toJson(DenormalisedTag(t))) } getOrElse NotFound
-      } recover {
-        commandErrorAsResult
+  def createTag = CORSable(conf.corsablePostDomains: _*) {
+    (APIAuthAction andThen CreateTagPermissionsCheck).async { req =>
+      implicit val username = Option(req.user.email)
```

Comment:
How would that look Luke? Do I simply remove the word `implicit` ?

Reply:
Also I see the whole file is done in this way... should I change all of them?

Reply:
After talking to @currysoup about this, have decided that while good practice, this improvment would be outside the scope of the PR since all the other methods on the `TagManagementApi` work in this way. So would require a fair amount of clean up